src/main/java/com/github/seratch/jslack/api/model/block/LayoutBlock.java

```diff
@@ -0,0 +1,9 @@
+package com.github.seratch.jslack.api.model.block;
+
+/**
+ * Base building block for Block Kit from Slack.
+ * 
+ * https://api.slack.com/reference/messaging/blocks
+ */
+public abstract class LayoutBlock {
```

Comment:
Any reasons to choose abstract class over interface here?

Reply:
I've toyed with both variants. Either would be possible. Using a base class would allow to move shared properties from subclasses to LayoutBlock (and BlockElement, and ...). However I did not decide yet, as I haven't worked with Lombok before and:
- did not know if the properties would get the proper builder support.
- the documentation on api.slack.com mentions all properties for each class, repeating the shared ones, so I figured that mirroring the documentation might be preferable over a minimalistic OO model

I decided for abstract base class because these blocks have an is-a relationship, not an implements. That said, I can be persuaded to change it to interfaces: I'm not married to them being in a class relationship.

Reply:
Thank you for taking time for the clear explanation. I personally prefer interfaces here. I appreciate it if you could respect my preference.

As far as I know, we can say the possible common attribute is only `type`. Therefore, I think the interface may be like this.
```java
public interface LayoutBlock {

    /**
     * The type of block.
     **/
    String getType();
}
```

Ideally, I would like to use the explanation in `slack-api-specs` project but [it seems that the project has not supported Block Kit yet](https://github.com/slackapi/slack-api-specs/issues/18
). So, we should use the explanation in [the documentation page](https://api.slack.com/block-kit).

Reply:
Fixed. I've also updated the javadoc to point to the Block Kit Guide and the Block Kit Reference